{"event": "verbose", "uuid": "415fc4e2-cec9-4db0-a63a-f9e93370c21c", "counter": 4, "stdout": "\u001b[0;31mThe error appears to be in '/home/vmadm/demo/RapidServe/southbound/deploy_con.yaml': line 77, column 7, but may\u001b[0m", "start_line": 3, "end_line": 4, "runner_ident": "15193494-cf9a-46ef-9aa9-78371df8d38b", "created": "2024-04-24T02:22:05.317774"}